README.md

```diff
@@ -360,3 +361,30 @@ In order to access to GCS, authentication credentials should be provided. More i
 #### GCS Testing
 
 The tests for `get_gcs.go` require you to have GCP credentials set in your environment.  These credentials can have any level of permissions to any project, they just need to exist.  This means setting `GOOGLE_APPLICATION_CREDENTIALS="~/path/to/credentials.json"` or `GOOGLE_CREDENTIALS="{stringified-credentials-json}"`.  Due to this configuration, `get_gcs_test.go` will fail for external contributors in CircleCI.
+
+### SMB (smb)
+
+To access samba shared folder, for Unix user it's necessary to have the [smbclient](https://www.samba.org/samba/docs/current/man-html/smbclient.1.html) installed or a local mount of the shared folder.
+For windows users, only the access permission is necessary.  
+
+Some examples for addressing the scheme:    
+- smb://username:password@host/shared/dir (downloads directory content)
+- smb://username@host/shared/dir
+- smb://host/shared/dir 
+- smb://username:password@host/shared/dir/file (downloads file)
+- smb://username@host/shared/dir/file
+- smb://host/shared/dir/file
+- //host/shared/dir (downloads directory content)
+- //host/shared/dir/file (downloads file)
+
+With and without the smb:// scheme, go-getter will first look for a local mount with file path as `/host/shared/dir` (Unix) or `//host/shared/dir` (Windows), and if the local mount is not available 
+it will then try to download the file/directory via `smbclient` command.  
```

Comment:
I'd may be try to regroup the two explanations together because my fear is that users could just read the first one and be done, I'll to regroup them to my understanding please feel free to fix it :D

```suggestion
The SMB getter will try to get files from samba when a url is prefixed with `smb://` for example `smb://foo/bar/dir`. 

The go-getter will try for a local mount of the path such as `/foo/bar/dir` (Unix) or `//foo/bar/dir` (Windows), and will fallback to using the [`smbclient`](https://www.samba.org/samba/docs/current/man-html/smbclient.1.html) command to download the file.

⚠️ On Unix when the smb share is not mounted, [`smbclient`](https://www.samba.org/samba/docs/current/man-html/smbclient.1.html) needs to be installed for this to work.

Some examples for addressing the scheme:    
- smb://username:password@host/shared/dir (downloads directory content)
- smb://username@host/shared/dir
- smb://host/shared/dir 
- smb://username:password@host/shared/dir/file (downloads file)
- smb://username@host/shared/dir/file
- smb://host/shared/dir/file
- //host/shared/dir (downloads directory content)
- //host/shared/dir/file (downloads file)

 
```

Reply:
Love it! Thanks!